app/src/main/java/com/gophillygo/app/activities/MapsActivity.java

```diff
@@ -85,16 +90,29 @@ protected void onCreate(Bundle savedInstanceState) {
     public void onMapReady(GoogleMap googleMap) {
         this.googleMap = googleMap;
         googleMap.getUiSettings().setMapToolbarEnabled(false);
-        panToCurrentLocation();
+        panToLocation();
     }
 
-    private void panToCurrentLocation() {
-        if (googleMap == null) {
+    private void panToLocation() {
+        if (googleMap == null || locationSet) {
```

Comment:
No I don't think so. The blue dot will move with the user location, which seems sufficient to me.

We don't want to let the user pan the map to a location they are interested in and then reset it back to the default view because they moved a few feet.

Additionally, we don't want to pan to the user's current location if the map was opened from a detail activity (instead we want to start centered on the attraction they were looking at).